code/app.js

```diff
@@ -0,0 +1,13 @@
+const eventType = 'click';
+const myButton = document.querySelector('button#randomize');
+
+const myCallback = function() {
+    console.log('The button was clicked!');
+ const newImg= document.createElement('img');
+newImg.setAttribute('src','https://source.unsplash.com/1600x900');
```

Comment:
for the bonus you can get the value of height and width from html and use it when writing the url 

```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");

 let url = ("https://source.unsplash.com/" + width.value + "x" + height.value);
newImg.serAttribute('src', url)

```